cmd/create/cluster/cmd.go

```diff
@@ -941,6 +941,97 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	var billingAccounts []string
+	cloudAccounts, err := r.OCMClient.GetBillingAccounts()
+
+	// get a list of CloudAccountID
+	for _, account := range cloudAccounts {
+		if account.CloudProviderID() == "aws" && !helper.HasPrefix(billingAccounts, account.CloudAccountID()) {
+			contracts, hasContract := account.GetContracts()
+			var contractString string
+			if hasContract {
+				dimensions := contracts[0].Dimensions()
+				for _, dimension := range dimensions {
+					if ocm.GetDimensionValue(dimension, "four_vcpu_hour") > 0 ||
+						ocm.GetDimensionValue(dimension, "control_plane") > 0 {
+						contractString = " [Contract enabled]"
+					}
+				}
+			}
+			billingAccounts = append(billingAccounts, account.CloudAccountID()+contractString)
+		}
+	}
+
+	if billingAccount == "" && !interactive.Enabled() {
+		if helper.HasPrefix(billingAccounts, awsCreator.AccountID) {
+			billingAccount = awsCreator.AccountID
+			r.Reporter.Infof("Using '%s' as billing account", billingAccount)
+			r.Reporter.Infof("To use a different billing account, add --billing-account xxxxxxxxxx to previous command")
+		}
```

Comment:
Question: Should we just error out here?
The billing account is empty and we can't find a default value and the user is not using interactive so we can't supply another billing account value.

Reply:
@mikeshng I think we are making it mandatory now

Reply:
Can you double check? If it's mandatory then I think you should do an else and err out here.

Reply:
Confirm with Lucas that we should allow empty => calls the backend and the backend will decide to error or not.